libs/commands/exec/src/command.ts

```diff
@@ -69,9 +70,8 @@ const command: CommandModule = {
 
     return filterOptions(yargs);
   },
-  handler(argv) {
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
-    return require(".")(argv);
+  async handler(argv) {
+    return factory(argv);
```

Comment:
I did it like that because there is a bug in used yargs version.
If the function is async and we use an import with await. Yargs controll flow continues and skips -- arguments before the command is invoked. With yargs 17.7.2(latest) it is working as expected. I will update yargs with a seperate change.